AI-accelerated-routing/initializing_cuopt_with_rl_solutions.ipynb

```diff
@@ -161,8 +161,34 @@
     "!pip install --upgrade --extra-index-url https://pypi.nvidia.com cuopt-cu12 nvidia-nvjitlink-cu12 rapids-logger==0.1.19\n",
     "\n",
     "# For cuda 13\n",
-    "# !pip install --upgrade --extra-index-url https://pypi.nvidia.com cuopt-cu13 nvidia-nvjitlink-cu13 rapids-logger==0.1.19 # For cuda 13\n",
-    "\n",
+    "# !pip install --upgrade --extra-index-url https://pypi.nvidia.com cuopt-cu13 nvidia-nvjitlink-cu13 rapids-logger==0.1.19 # For cuda 13"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d8351035",
+   "metadata": {},
+   "source": [
+    "### If Running in colab, please paste the setup code generated in launcher notebook in the below cell\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "89d7c3a7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# If Running in colab, please paste the setup code generated in launcher notebook in the below cell\n"
```

Comment:
Is this supposed to be a repeat of the comment above?

Reply:
Yeah, just added to code block as well just so users will not get confused